db/migrate/20220219083831_add_users_ref_to_posts.rb

```diff
@@ -1,5 +1,5 @@
 class AddUsersRefToPosts < ActiveRecord::Migration[7.0]
   def change
-    add_reference :posts, :user, null: false, foreign_key: foreign_key: {to_table: :users}
+    add_reference :posts, :user, null: false, foreign_key: true
   end
 end
```

Comment:
- [ ] I created some seed data but I got an error below when I tried seeding the data `rails db:seed`. I tried resolving the error by replacing line 3 from `add_reference :posts, :user, null: false, foreign_key: true` to `add_reference :posts, :author, foreign_key: { to_table: :users }` then I ran migration `rails db:migrate` and I was able to seed the data. This is because you're supposed to reference the author in the `users` table.

<img width="313" alt="Code_KCistDQYYw" src="https://user-images.githubusercontent.com/81468675/155500962-5b869a0c-9696-4970-95b9-f5835d98cbef.png">